hack/make-rules/test-cmd.sh

```diff
@@ -2313,8 +2313,9 @@ __EOF__
   kubectl-with-retry rollout resume deployment nginx "${kube_flags[@]}"
   # The resumed deployment can now be rolled back
   kubectl rollout undo deployment nginx "${kube_flags[@]}"
-  # Check that the new replica set (nginx-618515232) has all old revisions stored in an annotation
-  kubectl get rs nginx-618515232 -o yaml | grep "deployment.kubernetes.io/revision-history: 1,3"
```

Comment:
> Is the change to use versioned api going to change hashes in all pre-existing deployments?

Yes. I discussed with @janetkuo back then, IIRC, deployment controller shouldn't depend on the stable hash, because any changes to the API will alter the hash, like when we added ObjectMeta.OwnerReferences to the API.